internal/interpreter/api.rs

```diff
@@ -1159,13 +1164,33 @@ fn component_definition_properties2() {
 
     let props = comp_def.properties().collect::<Vec<(_, _)>>();
 
-    assert_eq!(props.len(), 1);
+    assert_eq!(props.len(), 2);
     assert_eq!(props[0].0, "sub-text");
     assert_eq!(props[0].1, ValueType::String);
+    assert_eq!(props[1].0, "xreadonly");
 
     let callbacks = comp_def.callbacks().collect::<Vec<_>>();
     assert_eq!(callbacks.len(), 1);
     assert_eq!(callbacks[0], "hello");
+
+    let instance = comp_def.create();
+    assert_eq!(
+        instance.set_property("xreadonly", SharedString::from("XXX").into()),
+        Err(SetPropertyError::AccessDenied)
+    );
+    assert_eq!(instance.get_property("xreadonly"), Ok(Value::String("the value".into())));
+    assert_eq!(instance.get_property("xreadonly"), Ok(Value::String("the value".into())));
```

Comment:
Just curious, why test the getter twice?

Reply:
bad copy paste